### DISCUSSION TITLE
Best way to dump model?

### DISCUSSION THREAD

**Discussion by alexandercommon** [LocalCascadeEnsemble/LCE#15] [General]
Before I dive in and try and figure it out myself, is there an easy way to dump the model similar to XGBoost? Links below.

Thanks! Loving it so far.

https://xgboost.readthedocs.io/en/stable/python/python_api.html#xgboost.Booster.dump_model
https://xgboost.readthedocs.io/en/stable/python/python_api.html#xgboost.Booster.get_dump

**Comment by Wuuzzaa** [comment]
AFAIK it is not implemented in LCE. Just use the good old dump and load functions from pythons [joblib]( https://joblib.readthedocs.io/en/latest/generated/joblib.dump.html#joblib.dump).
